scholarx/archive/functions.js

```diff
@@ -117,25 +97,25 @@ async function loadData() {
 }
 loadData();
 //this function run in every  render profiles functions to reset the page structure based on the selection
-function noRsultsPageHelper(){
+function noRsultsPageHelper() {
     $("#noResultsMessage").hide();
-    if(document.getElementById("selection").value === "mentors"){
+    if (document.getElementById('mentorsCheckbox').checked == true) {
         $("#showMentors").show();
-    }else if(document.getElementById("selection").value === "mentees"){
+    } else if (document.getElementById("selection").value === "mentees") {
```

Comment:
```suggestion
    } else if (document.getElementById('menteesCheckbox').checked == true) {
```
Also @YoshithaRathnayake you have to replace every these places in function.js

Reply:
Yeah I did @janithlahirukariyawasam searched and replaced in VS Code

Reply:
Now that problem is also fixed no?

Reply:
Good Job @YoshithaRathnayake , But we replaced the dropdown with checkboxes, so also the above places need to be changed, if not there can be issues when it working Eg: filtering issues like that . can you change every places like below
![image](https://github.com/sef-global/sef-site/assets/88109775/99f953df-9b5c-469a-9e05-b7bc24f4725d)

Reply:
Yeah I changed I searched in visual studio code and replaced them. Are there any codes that have not changed? @janithlahirukariyawasam

Reply:
Yeah I thought only mentors now I changed the mentees too @janithlahirukariyawasam check whether its ok now?